app/assets/stylesheets/application.css.scss

```diff
@@ -49,8 +49,12 @@
 }
 
 .text-white {
-  color: #ffffff;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.